assets/calc_fingerprint.sh

```diff
@@ -0,0 +1 @@
+(echo $@ && find $@ -type f -print0 | xargs -0 sha256sum) > $1.fingerprint
```

Comment:
1. A description of each step would be welcome
2. Such script should have shebang
```suggestion
#!/usr/bin/env bash
(echo $@ && find $@ -type f -print0 | xargs -0 sha256sum) > $1.fingerprint
```